lib/Mojolicious.pm

```diff
@@ -496,8 +512,7 @@ endpoints for your application.
   my $secrets = $app->secrets;
   $app        = $app->secrets([$bytes]);
 
-Secret passphrases used for signed cookies and the like, defaults to the L</"moniker"> of this application, which is
-not very secure, so you should change it!!! As long as you are using the insecure default there will be debug messages
+Secret passphrases used for signed cookies and the like, defaults to 44 random characters in the A..Za..z0..9 range. As long as you are using the insecure default there will be debug messages
```

Comment:
With this change there would no longer be an "insecure default" nor debug messages about such.

Reply:
This has been updated